Assets/BML/ScriptableObjectCore/Scripts/Variables/Variable.cs

```diff
@@ -3,9 +3,11 @@
 using BML.ScriptableObjectCore.Scripts.Utils;
 using Sirenix.OdinInspector;
 using Sirenix.Serialization;
-using UnityEditor;
 using UnityEngine;
 using UnityEngine.Serialization;
+#if UNITY_EDITOR
+using UnityEditor;
```

Comment:
Does this actually need to be conditional?

Reply:
good question, honestly not sure